cmd/kubeadm/app/phases/uploadconfig/uploadconfig.go

```diff
@@ -37,6 +37,53 @@ const (
 	NodesKubeadmConfigClusterRoleName = "kubeadm:nodes-kubeadm-config"
 )
 
+// UploadRemoveConfiguration uploads a new configuration with the clinet removed
+func UploadRemoveConfiguration(cfg *kubeadmapi.InitConfiguration, client clientset.Interface) error {
```

Comment:
Comment of function should say that nil is returned on success and the error if any on error.